components/Buttons/examples/ButtonsTypicalUseExampleViewController.m

```diff
@@ -23,6 +23,8 @@
 
 #import "supplemental/ButtonsTypicalUseSupplemental.h"
 
+const CGFloat kAllyButtonHeight = 48.0;
```

Comment:
Nit: "Ally" is confusing (I think I'm an ally to many). Perhaps `kMinimumAccessibleButtonHeight`?